torch/_refs/fft.py

```diff
@@ -78,7 +81,7 @@ def _maybe_promote_tensor_fft(
     new_type = _promote_type_fft(cur_type, require_complex)
     if cur_type == new_type:
         return t
-    return prims.convert_element_type(t, new_type)
+    return _maybe_convert_to_dtype(t, new_type)
```

Comment:
This conversion is known to occur -- the previous check handles the case where they're the same. `_maybe_convert_to_dtype` can be used here but it should replace the `if` above

Reply:
Agreed with you :) Updated it. Thanks